src/dotnet/UI.Blazor/Components/Avatar/AvatarName.razor

```diff
@@ -24,4 +27,6 @@
     [Parameter] public string Class { get; set; } = "";
     [Parameter] public EventCallback<MouseEventArgs> Click { get; set; }
     [Parameter] public bool IsAnonymous { get; set; }
+    [Parameter] public bool ShowOwnAsYou { get; set; }
```

Comment:
One parameter `ShowAsYou` is enough. Yoy don't need 2 parameters: ShowOwnAsYou and IsOwn